ambry-replication/src/main/java/com.github.ambry.replication/ReplicationManager.java

```diff
@@ -87,45 +88,53 @@ public RemoteReplicaInfo(ReplicaId replicaId, ReplicaId localReplicaId, Store lo
     initializeTokens(token);
   }
 
-  public ReplicaId getReplicaId() {
+  ReplicaId getReplicaId() {
     return replicaId;
   }
 
-  public ReplicaId getLocalReplicaId() {
+  ReplicaId getLocalReplicaId() {
     return localReplicaId;
   }
 
-  public Store getLocalStore() {
+  Store getLocalStore() {
     return localStore;
   }
 
-  public Port getPort() {
+  Port getPort() {
     return this.port;
   }
 
-  public long getReplicaLagInBytes() {
+  long getRemoteLagFromLocalInBytes() {
     if (localStore != null) {
       return this.localStore.getSizeInBytes() - this.totalBytesReadFromLocalStore;
     } else {
       return 0;
     }
   }
 
-  public FindToken getToken() {
+  long getLocalLagFromRemoteInBytes() {
```

Comment:
Where will the local lag from remote be used? Are we planning on adding a metric for this?